services/merge.ts

```diff
@@ -0,0 +1,221 @@
+    log('No conflicts found nothing to process', LOG_LEVEL.INFO);
+    return;
+  }
+
+  const batches = createBatchesForConflicts(conflicts, batchSize);
+  for (const batch of batches) {
+    log(`Starting on batch ${batches.indexOf(batch) + 1}/${batches.length}`);
+    const personsWithConflictAndFlag = await getConflictingPersonUris(batch);
+
+    const unresolvableConflicts = personsWithConflictAndFlag.filter(
+      (p: Conflict) => p.isUnresolvable,
+    );
+    log('Add flags to persons in conflict that are unresolvable');
+    await addIsConflictingFlagToPersons(unresolvableConflicts);
+
+    const resolvableConflicts = personsWithConflictAndFlag.filter(
+      (p: Conflict) => !p.isUnresolvable,
+    );
+    if (resolvableConflicts.length >= 1) {
+      log('Update usage as subject');
+      await updateConflictUsageToPersonAsSubject(resolvableConflicts);
+      log('Update usage as object');
+      await updateConflictUsageToPersonAsObject(resolvableConflicts);
+      log('Setup tombstones');
+      await setupTombstoneForConflicts(resolvableConflicts);
+    }
+  }
+}
+
+export async function addIsConflictingFlagToPersons(
+  conflicts: Array<Conflict>,
+): Promise<void> {
+  if (conflicts.length === 0) {
+    return;
+  }
+
+  const values = conflicts.map(
+    (conflict) =>
+      `( ${sparqlEscapeUri(conflict.conflictUri)} ${sparqlEscapeUri(conflict.personUri)})`,
+  );
+  const queryString = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX person: <http://www.w3.org/ns/person#>
+    PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+    PREFIX dct: <http://purl.org/dc/terms/>
+
+    INSERT {
+      GRAPH ?g {
+        ?conflict ext:conflictsWith ?person  .
+      }
+    }
+    WHERE {
+      VALUES ( ?conflict ?person) { ${values.join('\n')} }
+      GRAPH ?g {
+        ?conflict a person:Person .
+      }
+      ?g ext:ownedBy ?organization .
+    }
+  `;
+
+  try {
+    await updateSudo(queryString);
+  } catch (error) {
+    log(`Error was thrown on conflicts: ${JSON.stringify(conflicts)}`);
+    throw new Error(
+      `Something went wrong while adding ext:conflictsWith flag to ${conflicts.length} persons.`,
+    );
+  }
+}
+
+export async function updateConflictUsageToPersonAsSubject(
+  conflicts: Array<Conflict>,
+): Promise<void> {
+  if (conflicts.length === 0) {
+    return;
+  }
+
+  const values = conflicts.map(
+    (c) =>
+      `( ${sparqlEscapeUri(c.conflictUri)} ${sparqlEscapeUri(c.personUri)} )`,
+  );
+  const queryString = `
+    PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
+    PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+    PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX person: <http://www.w3.org/ns/person#>
+
+    DELETE{
+      GRAPH ?conflictG {
+        ?conflict dct:modified ?modified .
```

Comment:
you're deleting from conflict but inserting on person. So person now has 2 modifieds